doc/multiBand.dox

```diff
@@ -0,0 +1,73 @@
+namespace lsst { namespace pipe { namespace tasks {
+/**
+\page pipeTasks_multiBand High-level Overview of Multi-Band Coadd Processing
+
+The goal of coadd measurement is to consistently detect and measure sources across coadds constructed in 
+multiple filter bands in a consistent and reasonable manner. Specifically, we would like to measure the same 
+(deblended) sources across all photometric bands. There are several advantages to imposing this requirement. 
+We get the same deblending in all the bands. This allows us to run forced photometry on consistently deblended 
+objects which gives us reasonable colors. It also permits us to obtain measurements of flux in the case of 
+drop-out objects such as Lyman-break galaxies. Our strategy for meeting the consistent deblending requirement 
+can be broken up into a number of simple steps.
+<DL> 
+  <DT> i. \ref DetectCoaddSourcesTask_ "DetectCoaddSourcesTask": Detect Sources in Individual Coadds</DT> 
+  <DD> This step is the last step in the process of constructing a coadd and only temporarily belongs in the 
+  Multi-Band Coadd processing pipeline. After scaling the variance plane in a coadd to correct for the 
```

Comment:
I'm not sure what it means to say this "only temporarily belongs". Isn't a source detection step always necessary to start the ball rolling?

Also: I'd avoid using the term "pipeline". LDM-151 (the DM Applications Design) uses that term for some quite specific pieces of software, and this isn't one of them. Maybe prefer "procedure" or similar.